### PR TITLE
[UPDATE][1] updates to outline potential metadata updates

### PR DESCRIPTION
**Pull request by tea-mo903**
related to the issue. Just updated some doc in the README.MD in order to be compliant with related standards and potentials errors

### CODE REVIEW

**Comment by eternal-flame-AD** [comment]
I would say lets just provide an example to check for existing comments, and if one is found we append a tag to the existing block, otherwise we create a new metadata block for that. This would make more sense imo.

**Comment by tea-mo903** [comment]
yeah that makes sense to me too. Do you want me to update the PR or are you going to update it accordingly?

**Comment by tea-mo903** [comment]
I updated the PR, please review.

**Comment by tea-mo903** [comment]
Done. Thanks for the new go-flac release btw!

File: README.md [hunk 1]
```diff
@@ -40,21 +47,22 @@ import (
...
-			cmt, err = flacvorbis.ParseFromMetaDataBlock(*meta)
-			if err != nil {
-				panic(err)
-			}
-		}
-    }
-    return pic
+	cmts, idx := extractFLACComment(f)
+	if cmts == nil && idx > 0 {
+		cmts := flacvorbis.New()
```
Comment (eternal-flame-AD) [inline README.md]: Use `=` here, otherwise another scoped variable will be created and `cmts` will still be nil outside of the if statement

Comment (tea-mo903) [reply README.md]: True that. Updated accordingly

**Pull request merged**